udata/templates/reuse/mail_card.html

```diff
@@ -4,7 +4,7 @@
     <table height="94" cellpadding="0" cellspacing="0" style="margin: 0;padding: 0;width: 100%;border: 1px solid #eee;">
         <tr style="margin: 0;padding: 0;">
             <td width="98" style="margin: 0;padding: 0;">
-                <img src="{{ reuse.image(94, external=True) }}" style="margin: 0;padding: 0;width: 94px;height: 94px" width="94" height="94">
+                <img src="{{ reuse.image(94, external=True)|placeholder('reuse', True) }}" style="margin: 0;padding: 0;width: 94px;height: 94px" width="94" height="94">
```

Comment:
Sadly, it depends on the mail reader, some of them only interpret the width/height, others only interprets style, so both are required :/
Quite difficult to test, I didn't take the risk to break half of the clients.